lib/game.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'set'
+
+class Game
+  @wins = [[1, 2, 3], [1, 4, 7], [1, 5, 7], [2, 5, 8], [3, 6, 9], [3, 5, 7], [4, 5, 6], [7, 8, 9]]
```

Comment:
There's no need to repeat `@wins` or options here and then again in your `initialize` method. You can use `attr_writer`, `attr_reader` or `attr_accesor` here. You'd just use one of those three options depending if you just want to read the instance variable (getter), write it (setter) or both (getter and setter). So, you'd only do:

```
attr_accessor :options
attr_reader :wins
```

Read more about that [here](https://mixandgo.com/learn/ruby_attr_accessor_attr_reader_attr_writer).